supervisor/supervisorctl.py

```diff
@@ -190,20 +237,16 @@ def onecmd(self, line):
                             self.output('')
                             self.options.username = username
                             self.options.password = password
-                            return self.onecmd(origline)
+                            return self.onecmd_run(origline)
                         else:
-                            self.options.usage('Server requires authentication')
+                            self.handle_error('Server requires authentication')
```

Comment:
This might actually be

 self.handle_error('Server requires authentication', code=LSBInitErrorCode.INSUFFICIENT_PRIVLEDGES)